app/scripts/ui.js

```diff
@@ -3,6 +3,7 @@
 // This import sets up global functions required for Sentry to function.
 // It must be run first in case an error is thrown later during initialization.
 import './lib/setup-initial-state-hooks';
+import '../../development/wdyr';
```

Comment:
is it possible to import only when the environment variable is set rather than for all builds?

possibly adding the import to a separate script file and adding the condition to load the file in `app/scripts/app-init.js`

Reply:
it will be disabled for all builds by default